gpu4pyscf/grad/tests/test_uks_grad.py

```diff
@@ -26,40 +26,35 @@
 H       0.7570000000     0.0000000000    -0.4696000000
 '''
 
-pyscf_25 = version.parse(pyscf.__version__) <= version.parse('2.5.0')
-
 bas0='def2-tzvpp'
 grids_level = 5
 nlcgrids_level = 3
 def setUpModule():
     global mol_sph, mol_cart
-    mol_sph = pyscf.M(atom=atom, basis=bas0, max_memory=32000, charge=1, spin=1)
-    mol_sph.output = '/dev/null'
-    mol_sph.build()
-    mol_sph.verbose = 1
+    mol_sph = pyscf.M(atom=atom, basis=bas0, max_memory=32000,
+                      output='/dev/null', verbose=1)
 
-    mol_cart = pyscf.M(atom=atom, basis=bas0, max_memory=32000, charge=1, spin=1, cart=1)
-    mol_cart.output = '/dev/null'
-    mol_cart.build()
-    mol_cart.verbose = 1
+    mol_cart = pyscf.M(atom=atom, basis=bas0, max_memory=32000, cart=1,
+                       output='/dev/null', verbose=1)
 
 def tearDownModule():
     global mol_sph, mol_cart
     mol_sph.stdout.close()
     mol_cart.stdout.close()
     del mol_sph, mol_cart
 
+# FIXME: Why is the difference between CPU and GPU so large?
+# tol=1e-6 is not acceptable
```

Comment:
The consistency between CPU and GPU is at the level of 1e-10. We should be able to set it to 1e-8.

Reply:
Some tests show an error ~1e-6 . I don't think this is caused by bugs. I'm not clear the reason of the difference. They should match to a very high accuracy for such small systems. I will leave this comment there and further research on this problem.